### PR TITLE
Feature/advantages

### PR DESCRIPTION
**Pull request by Mrichk0**

### CODE REVIEW

File: src/sass/layouts/_advantages.scss [hunk 1]
```diff
@@ -0,0 +1,147 @@
+.advantages {
+  background-color: #fff5f6;
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss]: var

File: src/sass/layouts/_advantages.scss [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  padding-left: 20px;
+  padding-right: 70px;
+  @media screen and (min-width: 768px) {
+    display: flex;
+    padding-left: 35px;
+  }
+  @media screen and (min-width: 1280px) {
+    padding-left: 115px;
+    padding-right: 115px;
+  }
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss:15]: paddings goes to container

File: src/sass/layouts/_advantages.scss [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  margin-bottom: 20px;
+  font-family: Titan One;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 34px;
+  line-height: 39px;
+  text-align: left;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: #d41443;
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss]: var

File: src/sass/layouts/_advantages.scss [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+}
+.advantages__title {
+  list-style: none;
+  margin-bottom: 20px;
+  font-family: Titan One;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 34px;
+  line-height: 39px;
+  text-align: left;
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss]: no need, delete text-align

File: src/sass/layouts/_advantages.scss [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  //   background-image: url("../images/pnj/icon1\ \(2\).png");
+  // }
+}
+.advantages__title {
+  list-style: none;
+  margin-bottom: 20px;
+  font-family: Titan One;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 34px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss]: font-size: 36px

File: src/sass/layouts/_advantages.scss [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+  line-height: 39px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss]: line-height: 1.14;

File: src/sass/layouts/_advantages.scss [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    line-height: 66px;
+  }
+}
+.advantages__text {
+  font-family: DM Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 1.7;
+  color: #000000;
```
Comment (VovaRubakha) [inline src/sass/layouts/_advantages.scss]: var

File: src/partials/advantages.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<section class="advantages">
```
Comment (VovaRubakha) [inline src/partials/advantages.html:1]: add container

**Review by VovaRubakha — CHANGES_REQUESTED** [review]

**Comment by Mrichk0** [comment]
@VovaRubakha done

**Review by VovaRubakha — APPROVED** [review]

**Pull request merged**